File: 3206-find-common-elements-between-two-arrays/find-common-elements-between-two-arrays.py
class Solution:
    def findIntersectionValues(self, nums1: List[int], nums2: List[int]) -> List[int]:
        s1, s2 = set(nums1), set(nums2)

        a1 = sum(1 for i in nums1 if i in s2)
        a2 = sum(1 for i in nums2 if i in s1)

        return [a1, a2]

        

        
        
    

File: 3206-find-common-elements-between-two-arrays/find-common-elements-between-two-arrays.swift
class Solution {
    func findIntersectionValues(_ nums1: [Int], _ nums2: [Int]) -> [Int] {
        var arr: [Int] = []
        var answer1 = 0
        var answer2 = 0

        for i in nums1{
            for j in nums2{
                if i == j{
                    answer1 += 1
                    break
                }
            }
        }
        for j in nums2{
            for i in nums1{
                if i == j{
                    answer2 += 1
                    break
                }
            }
        }

        return [answer1, answer2]
        
    }
}